spec.bs

```diff
@@ -38,6 +38,11 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     text: height; url: embedded-content-other.html#attr-dim-height
 spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
   type: dfn
+    text: non-subresource requests; url: #non-subresource-request
+    text: navigation requests; url: #navigation-request
+    text: RequestDestination enum; url: #dom-requestdestination
```

Comment:
Can you just use the IDL syntax to grab the destination? https://tabatkins.github.io/bikeshed/#autolink-wrappers